azure_storage_client.go

```diff
@@ -279,7 +286,9 @@ func (util *snowflakeAzureClient) nativeDownloadFile(
 		blobClient = meta.mockAzureClient
 	}
 	if meta.options.GetFileToStream {
-		blobDownloadResponse, err := blobClient.DownloadStream(context.Background(), &azblob.DownloadStreamOptions{})
+		blobDownloadResponse, err := withCloudStorageTimeout(util.cfg, func(ctx context.Context) (azblob.DownloadStreamResponse, error) {
+			return blobClient.DownloadStream(context.Background(), &azblob.DownloadStreamOptions{})
```

Comment:
should it be `context` or rather `ctx`? (here and in couple different places)

Reply:
Good catch! I only found this and one another (s3 client) missing.